api/models/v0.3/modelAddress.js

```diff
@@ -151,44 +136,39 @@ class Address {
   }
 
   /**
-   * validationResponse(isWorkAddress)
-   * TODO: Need to call validation API to validate the address
-   * which depends on Service Objects.
-   *
-   * @param {boolean} isWorkAddress
+   * validateInAPI()
    */
-  validationResponse(isWorkAddress = undefined) {
-    // const { responses, validate } = AddressValidationAPI({ ilsClient });
-    // let validAddress = await validate(this.address);
-    // if false return null;
-    return true;
+  async validateInAPI() {
+    const { validate } = AddressValidationAPI({
+      soLicenseKey: this.soLicenseKey,
+    });
+
+    return await validate(this.address);
   }
 
   /**
-   * validatedVersion(isWorkAddress)
-   * Creates a validated address with updated attributes from
-   * the validation process.
-   *
-   * @param {boolean} isWorkAddress
+   * validate(isWorkAddress)
+   * Simple validation to make sure the address length is the proper length. If
+   * it is, it then validates the address in Service Objects.
    */
-  validatedVersion(isWorkAddress = undefined) {
+  async validate() {
+    const fullAddressLength = (this.address.line1 + this.address.line2).length;
+    if (fullAddressLength > 100) {
+      this.errors["line1"] =
+        "Address lines must be less than 100 characters combined.";
```

Comment:
Can we tell the user how many characters they're currently at?  (Also, aren't there any legitimate cases in which someone's address is over 100 characters...?)

Reply:
Yeah, it's possible to return how many characters they are over. There _could_ be cases there the length is per 100 but I'm not sure. This is an error that's carried over from the previous code. I think it makes some sense but I can't think of outliers right now.